src/quemb/kbe/pbe.py

```diff
@@ -129,6 +136,9 @@ def __init__(
         self.nproc = nproc
         self.ompnum = ompnum
 
+        self.use_cumulant = use_cumulant
```

Comment:
As we discussed in person @lweisburn , there are reasons to prefer having `use_cumulant` in the optimizer or the `oneshot` function